### DISCUSSION TITLE
Vercel Domain is pending verification

### DISCUSSION THREAD

**Discussion by sayedulkrm** [vercel/community#7383] [Account Issue] [answered]
### Question

I moved my project `i-steam` from my old Vercel account `creativelist` to a new account named `propaths`. I deleted the project and domain from the old account, but when I try to add `propathsports.com` to my new account, it says the domain is already in use. I followed the instructions to set a TXT record for `_vercel.propathsports.com` to verify ownership, but after 1 day, it still shows "Domain is pending verification".

Full Description:
Hi there, am having issue with my domian. I have a account name creativelist and there i setup a project name i-steam and added a domain name propathsports.com. Now i create a new vercel profile. And re deploy my project i-steam in my current account( which is this account. Name: "propaths"). 

The problem is am having issue to reusing the domain. Though i delete the project and domain from my old account (creativelist). When i about to add the domain it says this

```
Another Vercel account is using this domain. Set the following TXT record on _vercel.propathsports.com to use propathsports.com in this project. If you wish to transfer the entire domain, this will need to be done by the current domain owner. Once the verification is completed and the domain is successfully configured, the TXT record can be removed.
I did added the TXT and _vercel and the code in my "third party" Domain regestry. And its been 1 day, its is still showing that

```
It says on project

"Domain is pending verification"

![WhatsApp Image 2024-07-04 at 10 04 09 PM](https://github.com/vercel/community/assets/100830568/4e7ff311-37cf-489e-9485-1a03f7e187cd)

![WhatsApp Image 2024-07-04 at 10 04 33 PM](https://github.com/vercel/community/assets/100830568/aa306378-a428-4430-a8ec-cb8bd697b8f8)

**Comment by epleaner** [comment]
I am getting the same issue.

**Comment by nikkahel** [comment]
Same problem

**Comment by AlbertGithubHome** [comment]
I have access to my domain registrar, but I have encountered this domain name verification problem, need to get some help, the domain name verification needs to add the name of `_vercel` TXT record, but the domain name service provider we use JPdirect does not allow to add records containing the underline `"_"`, how to solve it? Are there other ways to verify the ownership of a domain name?

**Comment by sayedulkrm** [comment, marked as answer]
**Update**

After More than a week of waiting I had to reset my domain. And it solved the issue

[a comment by an automated account is not shown]
